meta/main.yml

```diff
@@ -4,14 +4,15 @@ galaxy_info:
   description: An Ansible role for installing and configuring a kafka cluster.
   company: Gigawatt
   license: BSD
-  min_ansible_version: 1.2
+  min_ansible_version: 2.0.0.2
   platforms:
   - name: Debian
     versions:
     - 7
   - name: Ubuntu
     versions:
     - trusty
+    - xenial
```

Comment:
we have used to role to successfully deploy kafka on xenial clusters (in addition to at least one trusty based one) 👍